heudiconv/utils.py

```diff
@@ -265,6 +265,32 @@ def json_dumps_pretty(j, indent=2, sort_keys=True):
     return js_
 
 
+def add_field_to_json(json_file, field_and_value):
+    """
+    Adds a given field (and its value) to a json file
+
+    Parameters:
+    -----------
+    json_file : str or Path
+        path for the corresponding json file
+    field_and_value : dict
+        pair of "key": "value" to add to the json file
+    """
+    for key, value in field_and_value.items():
+        lgr.debug(
+            'File: "{f}": Adding {k}: {v}'.format(
+                f=json_file,
+                k=key,
+                v=value,
+            )
+        )
+
+    with open(json_file) as f:
+        data = json.load(f)
+    data.update(field_and_value)
+    save_json(json_file, data, pretty=True)
```

Comment:
Unfortunately we aren't 100% consistent and use `pretty=True` only in  a few spots:
```shell
$> git grep -p pretty=True
heudiconv/bids.py=def populate_aggregated_jsons(path):
heudiconv/bids.py:        save_json(task_file, fields, sort_keys=True, pretty=True)
heudiconv/utils.py=def treat_infofile(filename):
heudiconv/utils.py:    save_json(filename, j_slim, sort_keys=True, pretty=True)

```
use of this function now needs to be aware of specifically to what "style" of json (indent and either pretty-fied) it conform, to ideally not change actual formatting of the file.  To at least allow to stay consistent, let's not hardcode `pretty` here and either make into

```suggestion
    save_json(json_file, data, pretty=pretty)
```
and add it as kwarg to signature (let's try to stay consistent with default indent and not allow to tune it unless will come needed)

Reply:
Addressed in c4a22240c4e5226a257ddc9248b3e6bf40e25280